Source/ACE/Entity/Container.cs

```diff
@@ -237,6 +237,32 @@ public virtual WorldObject GetInventoryItem(ObjectGuid objectGuid)
         }
 
         /// <summary>
+        /// This method is used to get all inventory items of Coin in this container (example of usage get all items of coin on player)
+        /// </summary>
+        /// <param name="objectGuid"></param>
```

Comment:
if you're not going to fill in param and returns, remove them.

Reply:
This should be fixed